test/utils/utils.go

```diff
@@ -128,6 +129,10 @@ func ValidateConditionsTrue(unstrObj *unstructured.Unstructured) error {
 			continue
 		}
 
+		if slices.Contains(excludedConditions, c.Type) {
```

Comment:
Maybe this would be a better fit for the [Go functional options pattern](https://golang.cafe/blog/golang-functional-options-pattern.html) since it's only used in this one instance, then each time `ValidateConditionsTrue` is used it wouldn't need to be passed an empty `[]string`